Frends.Community.RabbitMQ/RabbitMQTask.cs

```diff
@@ -86,33 +78,68 @@ public static bool WriteMessage([PropertyTab] WriteInputParams inputParams)
                     basicProperties = _channel.CreateBasicProperties();
                     basicProperties.Persistent = true;
                 }
-                
-                if (inputParams.WriteMessageCount != null &&
-                    _channel.MessageCount(inputParams.QueueName) >= int.Parse(inputParams.WriteMessageCount))
-                {
-                    _channel.TxSelect();
-                    _channel.ConfirmSelect();
-                    _channel.CreateBasicPublishBatch().Publish();
-                    _channel.TxCommit();
-                    if (_channel.MessageCount(inputParams.QueueName) > 0)
-                        _channel.TxRollback();
-                    return true;
-                }
 
-                else
+                
+                if (inputParams.WriteMessageCount == null || 
+                    inputParams.WriteMessageCount != null && int.Parse(inputParams.WriteMessageCount) == 1)
```

Comment:
The check for inputParams.WriteMessageCount != null is redundant as the previous condition inputParams.WriteMessageCount == null already verifies that. Also, maybe use writemessagecount <= 1 just to be safe